.github/CODEOWNERS

```diff
@@ -71,7 +71,7 @@
 /include/swift/Basic/                              @DougGregor
 /include/swift/Basic/Features.def                  @DougGregor @hborla
 /include/swift/ClangImporter                       @zoecarver @hyp @egorzhdan @beccadax @ian-twilightcoder @Xazax-hun
-/include/swift/DependencyScan                      @artemcm
+/include/swift/DependencyScan                      @artemcm @cachemeifyoucan
```

Comment:
Are you intentionally not adding this person to other DependencyScan patterns in this file?